Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -91,6 +91,7 @@ lemma commute_iff {R : Type*} [Mul R] [StarMul R] {x y : R}
   · simpa only [star_mul, hx.star_eq, hy.star_eq] using h.symm
 
 /-- Functions in a `StarHomClass` preserve self-adjoint elements. -/
+@[aesop 10% apply]
 theorem map {F R S : Type*} [Star R] [Star S] [FunLike F R S] [StarHomClass F R S]
```

Comment:
Shouldn't this work for `NonUnitalStarAlgHomClass`?

Reply:
Yes (except that class will cease to exist in about 15 minutes).